tests/test.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+for file in $PWD/*.c; do
```

Comment:
This will not work if executed from other dir.
Use the following instead:
```
for file in `dirname $0`/*.c; do
```

Reply:
Ok